readthedocs/rtd_tests/tests/test_builds.py

```diff
@@ -54,6 +57,165 @@ def test_build(self):
         self.assertRegexpMatches(cmd[0][0], r'python')
         self.assertRegexpMatches(cmd[0][1], r'sphinx-build')
 
+    def test_build_generate_index_file_force(self):
+        project = get(
+            Project,
+            slug='project-1',
+            documentation_type='mkdocs',
+            conf_py_file='',
+            enable_pdf_build=False,
+            enable_epub_build=False,
+            versions=[fixture()]
+        )
+        version = project.versions.all()[0]
+        build_env = LocalBuildEnvironment(
+            project=project,
+            version=version,
+            build={}
+        )
+        python_env = Virtualenv(version=version, build_env=build_env)
+        base_builder = MkdocsHTML(build_env, python_env)
+
+        def look_index_path(path):
```

Comment:
I know there is a function to fake paths based on a regex, but with this form looks more explicit, at least for me.